### PR TITLE
[13.0] [MIG] website_sale_product_detail_attribute_image: Migration to v13.0

### PR DESCRIPTION
**Pull request by CarlosRoca13**
cc @Tecnativa TT24152

### CODE REVIEW

**Review by sergio-teruel — APPROVED** [review]
Waiting to test in runbot

File: website_sale_product_detail_attribute_image/views/templates.xml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+<odoo>
+    <template
+        id="product_detail_attribute_image"
+        inherit_id="website_sale.product"
+        active="True"
+        customize_show="True"
+        name="Attribute's Images"
+    >
+        <xpath
+            expr="//div[@id='product_details']/../div[@class='col-md-6 col-xl-8']"
```
Comment (yajo) [inline website_sale_product_detail_attribute_image/views/templates.xml]: Reading runbot's logs you can find this warning:

```
2020-06-05 08:11:00,683 170 INFO openerp_test odoo.modules.loading: loading website_sale_product_detail_attribute_image/views/templates.xml
2020-06-05 08:11:00,702 170 WARNING openerp_test odoo.addons.base.models.ir_ui_view: Error-prone use of @class in view Attribute's Images (): use the hasclass(*classes) function to filter elements by their classes
```

Runbot gets an :x: if installing the module raises a warning. Here's the problem.  Fix:

```suggestion
            expr="//div[@id='product_details']/../div[hasclass('col-md-6', 'col-xl-8')]"
```

**Review by yajo — CHANGES_REQUESTED** [review]

File: website_sale_product_detail_attribute_image/views/templates.xml (hunk 1 above)
Comment (CarlosRoca13) [reply website_sale_product_detail_attribute_image/views/templates.xml]: Thanks for the help! I change it yet

File: website_sale_product_detail_attribute_image/README.rst [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+
+**Table of contents**
+
+.. contents::
+   :local:
+
+Usage
+=====
+
+* Go to 'Website > Settings > Configuration > Products > Attributes'.
```
Comment (HaraldPanten) [inline website_sale_product_detail_attribute_image/README.rst]: It think it's working as expected. Functional review 👍 

Just a minor change: Go to `Website > Configuration > Products > Attributes.`

**Review by HaraldPanten — APPROVED** [review]

**Review by yajo — APPROVED** [review]

**Pull request merged**